jykuo-love-shiritori/twp
# [frontend] preparations before calling api

Ex. Resolveeeeeees part of #94

### Changes


- [only the page of shops view of buyers](http://localhost:8080/sellerID/)


### Testing Approach (if unconventional)


please
1. `docker-compose up --build` 
2. login in with user account
3. go to [this page](http://localhost:8080/sellerID/) with browser

you will see something like this:
<img width="1680" alt="image" src="https://github.com/jykuo-love-shiritori/twp/assets/92158595/b361554c-5eef-4aad-bf6a-46959da97288">
or this:
<img width="1680" alt="image" src="https://github.com/jykuo-love-shiritori/twp/assets/92158595/11bce47e-f5b5-4e7c-b2fa-5de3371067c8">


### Additional context


it's normal that the broken images appear.
this pr is not for checking image is correct or not but the `CheckStatus` in /lib to let frontend team deal with the situation of calling api
